src/utils/hooks/useGetFullApplicationStructure.tsx

```diff
@@ -1,13 +1,20 @@
 import { useState, useEffect } from 'react'
-import { FullStructure } from '../types'
+import { FullStructure, PageNEW, SectionStateNEW } from '../types'
 import {
   ApplicationResponse,
   ApplicationStatus,
+  ReviewResponseDecision,
   useGetAllResponsesQuery,
 } from '../generated/graphql'
 import { useUserState } from '../../contexts/UserState'
 import { generateResponsesProgress } from '../helpers/structure/generateProgress'
 import addEvaluatedResponsesToStructure from '../helpers/structure/addEvaluatedResponsesToStructure'
+import addElementsById from '../helpers/structure/addElementsById'
+import deepEqual from 'deep-equal'
+import generateApplicantChangesRequestedProgress from '../helpers/structure/generateApplicantChangesRequestedProgress'
+import addSortedSectionsAndPages from '../helpers/structure/addSortedSectionsAndPages'
+import addApplicationResponses from '../helpers/structure/addApplicationResponses'
+import addApplicantChangeRequestStatusToElement from '../helpers/structure/addApplicantChangeRequestStatusToElement'
```

Comment:
Another possible good one to have an `index.ts` exporting all these files : )

Reply:
sorry, i should be more organised, i went on a rampage: cbd87a4da053d6db829a78665a4fe8deac766dc1